src-java/ping-topology/ping-messaging/src/main/java/org/openkilda/messaging/info/flow/HaFlowPingResponse.java

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.flow;
+
+import static org.openkilda.messaging.info.flow.FlowPingResponseUtils.checkPingSuccess;
+
+import org.openkilda.messaging.info.InfoData;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+@Data
+@Builder
+@AllArgsConstructor
+@EqualsAndHashCode(callSuper = false)
+@JsonNaming(SnakeCaseStrategy.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class HaFlowPingResponse extends InfoData {
+    @JsonProperty("ha_flow_id")
```

Comment:
@JsonProperty("ha_flow_id") is not necessary here. 
It was necessary for
```
    @JsonProperty("y_flow_id")
    String yFlowId; 
 ```
 because `SnakeCaseStrategy` transform `yFlowId` to `yflow_id`. (reason: there is only one character before capital F).
 
 But in haFolowId we have two characters before `F` for it's okay to remove @JsonProperty and keep @JsonNaming(SnakeCaseStrategy.class)

Reply:
Done. That's why copy-paste is not a good practice...